aws_lambda_powertools/utilities/validation/base.py

```diff
@@ -19,6 +26,10 @@ def validate_data_against_schema(data: Union[Dict, str], schema: Dict, formats:
         JSON Schema to validate against
     formats: Dict
         Custom formats containing a key (e.g. int64) and a value expressed as regex or callback returning bool
+    handlers: Dict
+        Custom methods to retrieve remote schemes, keyed off of URI scheme
+    provider_options: Dict
+        Arguments that will be passed directly to the underlying validate call
```

Comment:
suggestion: Until we have a providers implementation, add info that these are kwargs supported by `fastjsonschema` validate function, followed by a link to their API.

That will help customers know what's possible w/o opening the docs. 

As a rule of thumb, what we strive is to make sure a customer doesn't feel like "hmm I must be dumb as I don't know how to use this" <-- in short, reduce cognitive load, make the implicit explicit.

Reply:
Great catch.